src/ucp/core/ucp_ep.c

```diff
@@ -772,10 +772,10 @@ ucp_ep_local_disconnect_progress_remove_filter(const ucs_callbackq_elem_t *elem,
         return 0;
     }
 
-    /* we expect that only WIREUP_MSG/EP_REMOVED requests can be remained in the
-     * callback queue, because reply UCP EP created for sending this message is
-     * not exposed to a user */
-    ucs_assert(req->send.wireup.type == UCP_WIREUP_MSG_EP_REMOVED);
+    /* we expect that only EP flush request can be remained in the callback
+     * queue, because reply UCP EP created for sending WIREUP_MSG/EP_REMOVED
+     * message is not exposed to a user */
+    ucs_assert(req->send.uct.func == ucp_wireup_msg_progress);
```

Comment:
check also UCP_WIREUP_MSG_EP_REMOVED

Reply:
I now understand that it is not correct...
an unreleased request is a flush request
fixed